pages/index.tsx

```diff
@@ -7,10 +8,20 @@ import EventBanner from '../components/EventBanner';
 import HeroBanner from '../components/HeroBanner';
 import Footer from '../components/Footer';
 import Section from '../layouts/Section';
+import HackathonModal from '../components/HackathonModal';
+import Modal from 'react-modal';
+
+Modal.setAppElement('#__next'); // Set the app element for accessibility
 
 export default function Home() {
   const { sections } = home_page;
-  // UI
+
+  const renderSections = sections.map((section, i) => (
+    <section key={i}>
```

Comment:
in one of the other prs i removed the parent section here mind just doing that here as well. the `Section` component builds its own `section` so it doesnt make sense to have a section in a section.